bin/metrics-rabbitmq-queue.rb

```diff
@@ -120,16 +138,21 @@ def run
       queue['messages'] ||= 0
       drain_time = queue['messages'] / queue['backing_queue_status']['avg_egress_rate']
       drain_time = 0 if drain_time.nan? || drain_time.infinite? # 0 rate with 0 messages is 0 time to drain
-      output([config[:scheme], queue['name'], 'drain_time'].join('.'), drain_time.to_i, timestamp)
-
-      %w(messages consumers).each do |metric|
-        output([config[:scheme], queue['name'], metric].join('.'), queue[metric], timestamp)
+      queue['drain_time'] = drain_time.to_i
+
+      metrics = dotted_keys(queue)
+      metrics.each do |metric|
+        next unless metric.match(config[:metrics])
+        value = queue.dig(*metric.split('.'))
+        # Special case of *gress rates for backward-compatibility
```

Comment:
rather than using `*gress` can we use something like `(in|e)gress`? It took me a minute to not realize that this was not a typo.